src/includes/set-fingerprint/basic/php.mdx

```diff
@@ -0,0 +1,11 @@
+```php
+try {
+  // Run code that possibly throws an exception
+} catch (\Throwable $e) {
+  \Sentry\withScope(function (\Sentry\State\Scope $scope) use ($e) {
+    // Group errors together based on the message of the exception being thrown
```

Comment:
I still don't get this comment. There is no usage of the exception message whatsoever.

Reply:
🤦 hope to have fixed it now correctly!

Reply:
Perfextion